webapp/app/components/pages/home/HomePage.css

```diff
@@ -1,7 +1,15 @@
-.user-actions > button {
-  margin-right: 8px;
+table td, 
+table th {
+  overflow: hidden;
+  text-overflow: ellipsis;
 }
 
-.user-actions > button::last-child {
-  margin-right: 0;
+.user-list-action-buttons {
+  display: flex;
+  align-content: center;
+  align-items: flex-end;
 }
+
+.user-list-action-buttons button {
+  margin-left: 10px;
+}
```

Comment:
Add e new line here

Reply:
automatically formatted upon commit by pre commit hooks, end lines are removed  from css, only kept in javascript files